parm/post_avblflds.xml

```diff
@@ -1482,6 +1482,17 @@
        <scale>3.0</scale>
     </param>
 
+    <param>
```

Comment:
@ericaligo-NOAA Can you move this new entries of 508 after the definition of 506?